src/structure.js

```diff
@@ -81,6 +82,20 @@ function Structure (options) {
 inherits(Structure, EventEmitter);
 module.exports = Structure;
 
+Structure.prototype._subscribe = function(path, fn) {
```

Comment:
Yes, I've changed it little bit, but reference to parent object needs to be passed as this is mutating operation and listeners are immutable.